sdk/core/System.Net.ClientModel/api/System.Net.ClientModel.net6.0.cs

```diff
@@ -138,9 +142,12 @@ public abstract partial class MessageHeaders
     }
     public partial class MessagePipeline
     {
-        public MessagePipeline(System.Net.ClientModel.Core.PipelineTransport transport, System.ReadOnlyMemory<System.Net.ClientModel.Core.PipelinePolicy> policies) { }
-        public static System.Net.ClientModel.Core.MessagePipeline Create(System.Net.ClientModel.Core.PipelineOptions options) { throw null; }
-        public System.Net.ClientModel.Core.ClientMessage CreateMessage() { throw null; }
+        internal MessagePipeline() { }
+        public static void Create(System.Net.ClientModel.Core.PipelineOptions options, params System.Net.ClientModel.Core.PipelinePolicy[] perCallPolicies) { }
+        public static void Create(System.Net.ClientModel.Core.PipelineOptions options, System.ReadOnlySpan<System.Net.ClientModel.Core.PipelinePolicy> perCallPolicies, System.ReadOnlySpan<System.Net.ClientModel.Core.PipelinePolicy> perTryPolicies) { }
```

Comment:
I changed this so that MessagePipeline.Create actually does this pipeline freezing on options, rather than the call to pipelineOptions.GetPipeline().  This means that MessagePipeline has APIs for client-authors and PipelineOptions has APIs for client-users, and we don't have to EBN anything.

@KrzysztofCwalina, I'll be interested to get your thoughts on this approach at some point.

Reply:
Changed to something I'm happier with